tests/ociswrapper/ocis/ocis.go

```diff
@@ -74,7 +74,7 @@ func Start(envMap map[string]any) {
 func Stop() {
 	err := cmd.Process.Kill()
 	if err != nil {
-		log.Panic(err)
+		log.Println(err.Error())
```

Comment:
> https://github.com/owncloud/ocis/issues/6850#issuecomment-1685595647
> But maybe terminating the program is a good idea so that we know that something went wrong like in this case, we found the nil pointer issue with oCIS.

CC @phil-davis @2403905